tidb-cloud/migrate-from-aurora-bulk-import.md

```diff
@@ -5,34 +5,60 @@ summary: Learn how to migrate data from Amazon Aurora MySQL to TiDB Cloud in bul
+
+5. Check the cluster name and the region name in the **Target Cluster**.
+
+6. Modify the file patterns and add the table filter rules if needed. Click **Next**.
+
+    - **File Pattern**: modify the file pattern if you want to import files whose filenames match a certain pattern to a single target table.
+
+        > **Note:**
+        >
+        > When you use this feature, one import task can only import data to a single table at a time. If you want to use this feature to import data into different tables, you need to import several times, each time specifying a different target table.
+
+        To modify the file pattern, specify a custom mapping rule between CSV files and a single target table in the following fields:
+
+        - **File name**: enter a pattern that matches the names of the files to be imported. If you have one file only, enter the file name here directly.
+
+            For example:
 
-8. Click **Confirm**.
+            - `my-data?.sql`: all CSV files starting with `my-data` and one character (such as `my-data1.sql` and `my-data2.sql`) will be imported into the same target table.
+            - `my-data*.sql`: all CSV files starting with `my-data` will be imported into the same target table.
 
-    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
+        - **Target table**: enter the name of the target table in TiDB Cloud, which must be in the `${db_name}.${table_name}` format. For example, `mydb.mytable`. Note that this field only accepts one specific table name, so wildcards are not supported.
+
+    - **Table Filter**: If you want to filter which tables to be imported, you can specify one or more table filters in this field, separated by `,`.
+
+        For example:
+
+        - `db01.*`: all tables in the `db01` database will be imported.
+        - `db01.table01*,db01.table02*`: all tables starting with `table01` and `table02` in the `db01` database will be imported.
+        - `!db02.*`: except the tables in the `db02` database, all other tables will be imported. `!` is used to exclude tables that do not need to be imported.
+        - `*.*` : all tables will be imported.
+
+        For more information, see [table filter snytax](/table-filter.md#syntax).
+
+7. On the **Preview** page, confirm the data to be imported and then click **Start Import**.
+
+TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After the validation is completed and successful, the import task starts automatically. When the data import progress shows **Completed**, you have successfully imported the data to your database in TiDB Cloud.
```

Comment:
The paragraph is  used in the old import console. In the new import console ,the validation is after step 1 not step 3

Reply:
OK deleted this sentence: After the validation is completed and successful, the import task starts automatically.